content/departments/cloud/technical-docs/v2.0/index.md

```diff
@@ -72,7 +72,7 @@ All customer instances are considered part of the `prod` environment and all cha
 
 ##### [s2 instance](https://sourcegraph.sourcegraph.com/)
 
-This is the internal Cloud dogfood instance for the entire company. #dev-experience is responsible for rolling out nightly builds on this instance and #cloud is responsible for the maintenance of infrastructure, including Cloud SQL and underlying VM.
+This is the internal Cloud dogfood instance for the entire company. #devinfra is responsible for rolling out nightly builds on this instance and #cloud is responsible for the maintenance of infrastructure, including Cloud SQL and underlying VM.
```

Comment:
Historically DevX is the official owner of CD infra for S2. This is less true now because

- Cloud owns the upcoming Cloud release channel Job Fair project
- In the previous quarter, Cloud has been adding new features to the CD infra, e.g, continues upgrade that tracks specific release branch on another Cloud instance, cody-dev.sgdev.dev.

```suggestion
This is the internal Cloud dogfood instance for the entire company. #ask-cloud-ops is responsible for rolling out nightly builds on this instance. Additionally, they are responsible for the maintenance of infrastructure, including Cloud SQL and underlying VM.
```